client/src/components/errors/RtkErrorAlert.tsx

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright 2023 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import { SerializedError } from "@reduxjs/toolkit";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/dist/query";
+
+import { ErrorAlert } from "../Alert";
+
+interface RtkErrorAlertProps {
+  error: FetchBaseQueryError | SerializedError | undefined | null;
+}
+export function RtkErrorAlert({ error }: RtkErrorAlertProps) {
+  // ? REF: https://redux-toolkit.js.org/rtk-query/usage-with-typescript#type-safe-error-handling
+  if (error == null || error === undefined) return null;
+
+  // code or status
+  let errorCode = "Unknown";
+  if ("status" in error) errorCode = error.status.toString();
+  else if ("code" in error && error.code !== undefined)
+    errorCode = error.code.toString();
```

Comment:
General code style comment: prefer to use `const` whenever possible.

This prevents assigning another value to the variable unintentionally.

```suggestion
  const errorCode = "status" in error ? error.status.toString() : "code" in error && error.code !== undefined ? error.code.toString() : "Unknown";
```

Reply:
This makes sense, but I find nested ternary operators hard to interpret

Reply:
I'll go for you solution since it's safer